scripts/superglue-baselines.sh

```diff
@@ -18,7 +18,11 @@ function copa() {
 }
 
 function multirc() {
-    python main.py --config config/superglue-bert.conf --overrides "random_seed = ${seed}, cuda = ${gpuid}, run_name = multirc, pretrain_tasks = \"multirc\", target_tasks = \"multirc\", do_pretrain = 1, do_target_task_training = 0, do_full_eval = 1, batch_size = 4, val_interval = 1000"
+    python main.py --config config/superglue-bert.conf --overrides "random_seed = ${seed}, cuda = ${gpuid}, run_name = multirc, pretrain_tasks = \"multirc\", target_tasks = \"multirc\", do_pretrain = 1, do_target_task_training = 0, do_full_eval = 1, batch_size = 4, val_interval = 1000, val_data_limit = -1"
```

Comment:
is 5k (default val data limit) vs. 10k really crucial during training?

Reply:
Strictly speaking, no, but it's useful. Evaluation/validation takes a while because there is a lot of data and 5k batches is only ~1% of the training data, so this greatly speeds things up.

Reply:
(note: this comment applies below)